.golangci.yml

```diff
@@ -104,7 +104,6 @@ linters-settings:
     include-go-root: true
     packages-with-error-message:
       # See https://github.com/open-telemetry/opentelemetry-collector/issues/5200 for rationale
```

Comment:
```suggestion
```

This comment would be misleading if not removed.

Reply:
removed that comment